### ISSUE TITLE
[Feature] Add "Edit Config.yml" button to settings area

### ISSUE THREAD

**Issue by jeremymeyers** [stashapp/stash#1173]
As there are a bunch of features not surfaced into the configuration menus, a button to launch config.yml in the users default editor would be helpful (probably with a warning about "advanced settings that could break your install".

**Comment by WithoutPants** [comment]
The fact that stash is a webapp makes this not really feasible. There's no guarantee that the client is the same machine as the server (and in many cases won't be). Perhaps an in-app editor would be more suitable (like the custom css editor)?

**Comment by jeremymeyers** [comment]
yeah i was kinda thinking that? though i think at some point that might tie into a discussion about reworking the layout of the settings page because the Configuration tab is messy and not optimally organized

**Comment by jeremymeyers** [comment]
@WithoutPants what about this in an "advanced" submenu of config?
